### PR TITLE
VAGOV-000: Changing banner field help text and label.

### PR DESCRIPTION
**Pull request by ethanteague**
Changing field label and help text.

### CODE REVIEW

**Comment by kevwalsh** [comment]
@ethanteague behat failure, drupal spec tool i assume.

File: config/sync/field.field.node.full_width_banner_alert.field_alert_inheritance_subpages.yml [hunk 1]
```diff
@@ -9,15 +9,15 @@ id: node.full_width_banner_alert.field_alert_inheritance_subpages
 field_name: field_alert_inheritance_subpages
 entity_type: node
 bundle: full_width_banner_alert
-label: 'Inherited by subpages'
-description: 'Should all pages of the VAMC display this block? For example, if you pick VA Pittsburgh health care, should va.gov/pittsburgh-health-care/about-us, and all other children pages, display this alert? '
+label: 'Only show on VAMC System Page & Operating Status Page'
+description: 'By checking this box, you will limit the alert display to only the VAMC System Page.'
```
Comment (kevwalsh) [inline config/sync/field.field.node.full_width_banner_alert.field_alert_inheritance_subpages.yml]: this should read ' and the Operating status page."

Comment (ethanteague) [reply config/sync/field.field.node.full_width_banner_alert.field_alert_inheritance_subpages.yml]: :+1: updated @kevwalsh

**Comment by ethanteague** [comment]
> @ethanteague behat failure, drupal spec tool i assume.

:+1: updated @kevwalsh

File: config/sync/field.field.node.full_width_banner_alert.field_alert_inheritance_subpages.yml [hunk 2]
```diff
@@ -9,15 +9,15 @@ id: node.full_width_banner_alert.field_alert_inheritance_subpages
 field_name: field_alert_inheritance_subpages
 entity_type: node
 bundle: full_width_banner_alert
-label: 'Inherited by subpages'
-description: 'Should all pages of the VAMC display this block? For example, if you pick VA Pittsburgh health care, should va.gov/pittsburgh-health-care/about-us, and all other children pages, display this alert? '
+label: 'Only show on VAMC System Page & Operating Status Page'
+description: 'By checking this box, you will limit the alert to display only on the VAMC System Page and the Operating status page.'
```
Comment (swirtSJW) [inline config/sync/field.field.node.full_width_banner_alert.field_alert_inheritance_subpages.yml]: Inconsistent capitalization 
```suggestion
description: 'By checking this box, you will limit the alert to display only on the VAMC System Page and the Operating Status Page.'
```

Comment (ethanteague) [reply config/sync/field.field.node.full_width_banner_alert.field_alert_inheritance_subpages.yml]: Updated, @swirtSJW

**Pull request merged**